lxd/instance/drivers/driver_qemu.go

```diff
@@ -34,7 +34,7 @@ import (
 	"golang.org/x/sys/unix"
 	"gopkg.in/yaml.v2"
 
-	"github.com/lxc/lxd/client"
+	lxd "github.com/lxc/lxd/client"
```

Comment:
This shouldnt be changed. Please change your editor to not do this automatically.